assets/js/src/newsletters/send.jsx

```diff
@@ -478,10 +477,6 @@ class NewsletterSend extends React.Component {
           onChange={this.handleFormChange}
           onSubmit={this.handleSave}
         >
-          <InvalidMssKeyNotice
```

Comment:
This duplicated notice was not part of the issue but saw it during the development.
<img width="1434" alt="DuplicatedAllSending" src="https://user-images.githubusercontent.com/8002881/129365340-34201faf-8c43-4cf1-a1c8-b63dd0f4b218.png">